lib/cache/access_list_test.go

```diff
@@ -342,3 +342,32 @@ func TestCountAccessListMembersScoping(t *testing.T) {
 		}
 	}
 }
+
+func TestGetAllAccessListMembers(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	p := newTestPack(t, ForAuth)
+	t.Cleanup(p.Close)
+
+	clock := clockwork.NewFakeClock()
+
+	_, members, err := p.accessLists.UpsertAccessListWithMembers(context.Background(), newAccessList(t, "access-list", clock),
+		makeMembers(t, "access-list", 10),
+	)
+	require.NoError(t, err)
+
+	require.EventuallyWithT(t, func(t *assert.CollectT) {
+		result, err := stream.Collect(clientutils.ResourcesWithPageSize(ctx, p.cache.ListAllAccessListMembers, 1))
+		require.NoError(t, err)
+		require.Len(t, result, len(members))
+	}, time.Second*2, time.Millisecond*30)
```

Comment:
Nit: I'd increase to 10 sec right away.
```suggestion
	}, time.Second*10, time.Millisecond*30)
```